backend/src/api/mutation.rs

```diff
@@ -285,6 +291,39 @@ impl Mutation {
             target_realm
         };
 
+        let series = if let Some(current_realm_path) = current_realm_path {
+            let old_realm = Realm::load_by_path(current_realm_path, context)
+                .await?
+                .ok_or_else(|| invalid_input!("`currentRealmPath` does not refer to a valid realm"))?;
+            let blocks = BlockValue::load_for_realm(old_realm.key, context).await?;
+            if blocks.len() > 1 {
+                // This is already checked and prohibited on the front end side (admin UI),
+                // but I suppose checking here doesn't hurt .
```

Comment:
"doesn't hurt" -> is even required :D only frontend checking is no bueno. Also there is a space before `.`